navigation-testing/src/androidMain/kotlin/com/freeletics/khonshu/navigation/NavigatorTurbine.kt

```diff
@@ -198,7 +197,7 @@ public interface NavigatorTurbine {
      *
```

Comment:
nit: description in ln 195 needs to be adapted too

Reply:
https://github.com/freeletics/khonshu/pull/922/commits/4b3383c5db9595f230ae8781bf2a71f2cefa2001